Base_Scripts/Scrapers/template/list_pdf_extractor/list_pdf_extractor_v2.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import os
+from bs4 import BeautifulSoup
+import urllib
+import re
+import time
+import sys
+import configs
+from pathlib import Path
+
+p = Path(__file__).resolve().parents[3]
+sys.path.insert(1, str(p) + "/common")
+from bs_scrapers.get_files import get_files
+from bs_scrapers.extract_info import extract_info
+
+cur_dir = os.getcwd()
```

Comment:
You can should be able to replace these lines with `save_dir = "./data/"`

Reply:
Updated in fcf0ffdabc74acb7643fe6e23c088e1f3787d2f9